src/app/core/layers/geolayermixin.js

```diff
@@ -103,9 +97,9 @@ proto.getOlSelectionFeatures = function(){
   return this.olSelectionFeatures;
 };
 
-proto.addOlSelectionFeature = function({id, geometry}={}){
+proto.addOlSelectionFeature = function({id, feature}={}){
```

Comment:
@volterra79 the same goes for other this function, for example:

https://github.com/g3w-suite/g3w-client/blob/6e5b47a672ed6bd62cbc6f92f515c3effc1639c5/src/app/gui/table/tableservice.js#L160-L163

https://github.com/g3w-suite/g3w-client/blob/6e5b47a672ed6bd62cbc6f92f515c3effc1639c5/src/app/gui/table/tableservice.js#L253-L256

https://github.com/g3w-suite/g3w-client/blob/6e5b47a672ed6bd62cbc6f92f515c3effc1639c5/src/app/gui/table/tableservice.js#L160-L163

Reply:
It used by queryresultservice.js when click on selection tool